### PR TITLE
update to latest cuddle

### PR DESCRIPTION
**Pull request by adin234**

### CODE REVIEW

File: lib/YoukuAuth.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-
 Object.defineProperty(exports, "__esModule", {
     value: true
 });
 
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
```
Comment (pandafox) [inline lib/YoukuAuth.js:7]: What's changed here?

**Comment by pandafox** [comment]
Changes doesn't seem to relate to cuddle at all?

File: lib/YoukuAuth.js (hunk 1 above)
Comment (adin234) [inline lib/YoukuAuth.js:7]: this is generated by babel

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/anyTV/anytv-node-youku-uploader",
   "dependencies": {
-    "cuddle": "0.0.52",
+    "cuddle": "^1.0.3",
```
Comment (adin234) [inline package.json:21]: this is the only thing that was updated

File: lib/YoukuAuth.js (hunk 1 above)
Comment (pandafox) [inline lib/YoukuAuth.js:7]: Hum.. I'd rather not have generated code checked in. But I guess this looks similar enough to the previous item, I think this is okay.

**Comment by pandafox** [comment]
:star:

**Comment by adin234** [comment]
@pandafox i added a commit to bump the version to match the published one https://www.npmjs.com/package/anytv-node-youku-uploader

**Pull request merged**